bartop-ui/src/components/Auth/Auth.test.js

```diff
@@ -20,12 +23,23 @@ const requiredProps = {
   }
 };
 
+// mock the auth status for AuthButton
+const store = configureMockStore()({
+  authentication: {
+    status: mockAuthStatuses.default
+  }
+});
+
 const mountRouterAtLocation = location =>
   mount(
     <StaticRouter context={{}} location={location}>
       <Route
         path="/auth"
-        render={({ match }) => <Auth {...requiredProps} match={match} />}
+        render={({ match }) => (
+          <Provider store={store}>
```

Comment:
Because the `AuthButton` is rendered in the `Header`, and the `AuthButton` is connected to `Redux`, we need to mock the relevant state for tests